x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/ransomware_detection.cy.ts

```diff
@@ -24,33 +25,30 @@ describe('Ransomware Detection Alerts', { tags: ['@ess', '@serverless'] }, () =>
     });
   });
 
-  describe('Ransomware display in Alerts Section', () => {
+  after(() => {
+    cy.task('esArchiverUnload', 'ransomware_detection');
+  });
+
+  describe('Ransomware display on Alerts Page', () => {
     beforeEach(() => {
       login();
       visitWithTimeRange(ALERTS_URL);
       waitForAlertsToPopulate();
     });
 
-    describe('Alerts table', () => {
-      it('shows Ransomware Alerts', () => {
-        cy.get(ALERT_RULE_NAME).should('have.text', 'Ransomware Detection Alert');
-      });
+    it('should show Ransomware Alerts in alerts table', () => {
```

Comment:
what do you think about merging this test with the next? We could name it `should show Ransomware Alerts in alerts page` and inside you'd have a `cy.log('Ransomware Alerts in the alerts table')` then a couple of lines below another one `cy.log('Ransomware Alerts in the trend chart')`